components/formstack/sources/new-form/new-form.js

```diff
@@ -0,0 +1,34 @@
+const formstack = require("../../formstack.app.js");
+
+module.exports = {
+  key: "formstack-new-form",
+  name: "New Form (Instant)",
+  description: "Emits an event for each new form added.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    formstack,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+  },
+  async run() {
+    const formIds = this.db.get("formIds") || [];
+    const forms = await this.formstack.getForms();
```

Comment:
getting the entire list of forms for the account every 15 mins seems really inefficient.  We'll end up with a bunch of duplicates that could be avoided. you could save the pages you've iterated through and start from that.  Things could get missed if a form in a previous page has been deleted though.

Assuming that the fromstack api returns everything in id order, you could page through everything and only emit ids > the largest id emitted from the previous run.  Then save the greatest id emitted in the current run for the next run.